R/pkg/R/DataFrame.R

```diff
@@ -3249,9 +3249,12 @@ setMethod("as.data.frame",
 #' @note attach since 1.6.0
 setMethod("attach",
           signature(what = "SparkDataFrame"),
-          function(what, pos = 2, name = deparse(substitute(what)), warn.conflicts = TRUE) {
-            newEnv <- assignNewEnv(what)
-            attach(newEnv, pos = pos, name = name, warn.conflicts = warn.conflicts)
+          function(what, pos = 2L, name = deparse(substitute(what), backtick = FALSE),
+                   warn.conflicts = TRUE) {
```

Comment:
changing this to match the latest R-devel param signature

Reply:
So this is the only thing we might need to update if R changes something ? Also how does this keep working with the older R version as well

Reply:
this will need to be updated manually...